src/irsForms/ScheduleEIC.test.ts

```diff
@@ -0,0 +1,52 @@
+import { waitFor } from '@testing-library/react'
+import { FilingStatus, PersonRole, TaxesState } from '../redux/data'
+import ScheduleEIC from './ScheduleEIC'
+import F1040 from './F1040'
+
+afterEach(async () => {
+  await waitFor(() => localStorage.clear())
+  jest.resetAllMocks()
```

Comment:
If you reset mocks after each then I think your redux persist mock will go away after the first 'it' test. If another test is added that might be a problem

Reply:
Is that not desired? I don't think the second test should depend on the state resulting after the first test?

Reply:
You're right, I was thinking of `jest.restoreAllMocks`

Reply:
I get your point now, yea per resetAllMocks doc:    
```
     * Resets the state of all mocks.
     * Equivalent to calling .mockReset() on every mocked function.
```
sounds ok